engines/scumm/scumm-md5.h

```diff
@@ -186,7 +186,7 @@ static const MD5Table md5table[] = {
 	{ "37f56ceb13e401a7ac7d9e6b37fecaf7", "loom", "EGA", "EGA", 5748, Common::EN_ANY, Common::kPlatformDOS },
 	{ "37ff1b308999c4cca7319edfcc1280a0", "puttputt", "HE 70", "Demo", 8269, Common::EN_ANY, Common::kPlatformWindows },
 	{ "3824e60cdf639d22f6df92a03dc4b131", "fbear", "HE 62", "", 7732, Common::EN_ANY, Common::kPlatformDOS },
-	{ "387a544b8b10b26912d8413bab63a853", "monkey2", "", "Demo", 9770, Common::EN_ANY, Common::kPlatformDOS },
+	{ "387a544b8b10b26912d8413bab63a853", "monkey2", "Demo", "Demo", 9770, Common::EN_ANY, Common::kPlatformDOS },
```

Comment:
@AndywinXp I believe that `devtools/scumm-md5.txt` should always be updated first, or your scumm-md5.h change will be lost the next time someone runs the `devtools/md5table` tool again.